utils/checks.stanza

```diff
@@ -167,7 +170,7 @@ public pcb-check io (module:JITXObject) :
               ; println("checking %_ vs %_" % [ref(out-p) ref(in-p)])
               check-single-push-pull-levels(out-p, in-p)
 
-        if o is OpenCollector :
+        match(o:OpenCollector) :
```

Comment:
Using a `match` statement to cast `o` to an `OpenCollector`.

Reply:
Nice!